exercises/queen-attack/canonical-data.json

```diff
@@ -18,39 +18,54 @@
           "description": "queen with a valid position",
           "property": "create",
           "queen": {
-            "position": "(2,2)"
+            "position": {
+              "row": 2,
+              "column": 2
+            }
           },
           "expected": 0
```

Comment:
a little awkward that these are 0/-1 instead of just true/false. Probably worth fixing later on.

Reply:
Right now it's using 0/1 as false/true and -1 as error. It would probably be better to use the accepted `"error"` methodology, but that's probably worth its own pull request. Should one of us open an issue?

Reply:
I agree that a separate PR for this is a good idea.